src/components/cc-stretch-to-max-content/cc-stretch-to-max-content.js

```diff
@@ -0,0 +1,110 @@
+import { css, html, LitElement } from 'lit';
+
+/**
+ * A layout component used to switch visibility between several elements while preventing layout shifts.
+ *
+ * Use `visibleElementId` prop to provide the `id` value corresponding to the element you want to make visible.
+ * Other elements are hidden with `visibility: hidden`.
+ *
+ * You may center the content with the `centerContentHorizontally` & `centerContentVertically` props.
+ * You may conditionally disable the stretching with the `disableStretching` prop.
+ *
+ * More information about the issues this component solves and the technique it uses can be found in [Hubert Sablonnière's blog Post - "Prevent layout shifts with CSS grid stacks"](https://www.hsablonniere.com/prevent-layout-shifts-with-css-grid-stacks--qcj5jo/).
+ *
+ * @cssdisplay inline-grid
+ *
+ * @slot - The containers you want to switch between. The container you want to be displayed must have an `id` attribute matching the value of the `visibleElementId` prop.
+ */
+
+export class CcStretchToMaxContent extends LitElement {
+
+  static get properties () {
+    return {
+      centerContentHorizontally: { type: Boolean, attribute: 'center-content-horizontally', reflect: true },
+      centerContentVertically: { type: Boolean, attribute: 'center-content-vertically', reflect: true },
```

Comment:
could also provide a `center-content` property in addition to these ? (just thought of that while implementing the two attributes in `cc-orga-member` context...)